docker/generic/start_proxy.py

```diff
@@ -407,6 +407,23 @@ def make_argparser():
         Set the retry times for service control Report request.
         Must be >= 0 and the default is 5 if not set.
         ''')
+    parser.add_argument(
+        '--access_log',
```

Comment:
does "/dev/stdout" work for access_log?

If so,  should we default to /dev/stdout so that Envoy access log will show up in the stack driver logs.  I know it is showing something when deployed in Cloud Run, but I guess it is not from Envoy.   I am wondering if it shows anything when deployed in GKE.

Reply:
Yes, it works. Those should be the request log from [Cloud Run](https://cloud.google.com/run/docs/logging?hl=en) but they are very simple [ones](https://screenshot.googleplex.com/OYNUNKxPbWn).

I think we need be careful about this since:
- it will make duplicate log entries on Cloud Run
- the report logs in endpoints basically cover most needs though this access_log can log the headers you want which is cool

Yeah, we can think about it later.

Reply:
Agreed that this is basically a duplication of our SC Reports...